barretenberg/cpp/src/barretenberg/commitment_schemes/claim_batcher.hpp

```diff
@@ -122,7 +147,21 @@ template <typename Curve> struct ClaimBatcher_ {
         if (right_shifted_by_k) {
             aggregate_claim_data_and_update_batched_evaluation(*right_shifted_by_k, rho_power);
         }
+        if (interleaved) {
```

Comment:
it's technically very easy to support also the scenario when the interleaved group is odd (we'd have to compute r^{group size} and (-r)^{group_size}) but it seemed unnecessary because the translator has a certain choice of group size (power of two) also for efficiency